collector/mssql.go

```diff
@@ -246,1021 +244,1026 @@ type MSSQLCollector struct {
+			"(DatabaseReplica.TotalLogrequiringundo)",
+			[]string{"instance", "replica"},
 			nil,
 		),
 		TransactionDelay: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "transaction_delay"),
-			"(TransactionDelay)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "dbreplica_transaction_delay"),
+			"(DatabaseReplica.TransactionDelay)",
+			[]string{"instance", "replica"},
 			nil,
 		),
 
 		// Win32_PerfRawData_{instance}_SQLServerDatabases
 		ActiveTransactions: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "active_transactions"),
-			"(ActiveTransactions)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_active_transactions"),
+			"(Databases.ActiveTransactions)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		AvgDistFromEOLLPRequest: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "avg_dist_from_eollp_request"),
-			"(AvgDistFromEOLLPRequest)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_avg_dist_from_eollp_request"),
+			"(Databases.AvgDistFromEOLLPRequest)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		BackupPerRestoreThroughputPersec: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "backup_per_restore_throughput_persec"),
-			"(BackupPerRestoreThroughputPersec)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_backup_per_restore_throughput"),
+			"(Databases.BackupPerRestoreThroughputPersec)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		BulkCopyRowsPersec: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "bulk_copy_rows_persec"),
-			"(BulkCopyRowsPersec)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_bulk_copy_rows"),
+			"(Databases.BulkCopyRowsPersec)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		BulkCopyThroughputPersec: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "bulk_copy_throughput_persec"),
-			"(BulkCopyThroughputPersec)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_bulk_copy_throughput"),
+			"(Databases.BulkCopyThroughputPersec)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		Committableentries: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "committableentries"),
-			"(Committableentries)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_commit_table_entries"),
+			"(Databases.Committableentries)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		DataFilesSizeKB: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "data_files_size_kb"),
-			"(DataFilesSizeKB)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_data_files_size"),
```

Comment:
Sorry for not being clearer about this... The convention is to keep the unit in the name, so `_bytes` on all of these

Reply:
done